examples/cloudml-document-ai-patents/get_service_acct.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
In general, check this file against the [shell styleguide](https://google.github.io/styleguide/shell.xml)